filebeat/docs/inputs/input-common-file-options.asciidoc

```diff
@@ -397,3 +401,42 @@ file that hasn't been harvested for a longer period of time.
 This configuration option applies per input. You can use this option to
 indirectly set higher priorities on certain inputs by assigning a higher
 limit of harvesters.
+
+[float]
+===== `file_identity`
+
+Different `file_identity` methods can be configured to suit the
+environment you are collecting log messages from.
+
+
+*`inode_deviceid`*:: The default behaviour of {beatname_uc} is to differentiate
+between files using their inodes and device ids.
+
+[source,yaml]
+----
+file_identity.inode_deviceid: ~
+----
+
+*`path`*:: To identify files based on their paths use this strategy.
+
+WARNING: Only use this strategy if your log files are rotated to a folder
+outside of the scope of your input or not at all. Otherwise you end up
+with duplicated events.
+
+WARNING: This strategy does not support renaming files.
+If an input file is renamed, {beatname_uc} will read it again if the new path
+matches the settings of the input.
+
+[source,yaml]
+----
+file_identity.path: ~
+----
+
+*`inode_marker`*:: If the device id changes from time to time, you must use
+this method to tell apart files. Set the location of the marker file the following way:
```

Comment:
`tell apart` isn't great wording -- if only because it's unclear what it means.

Is it "in order to distinguish volumes"?